keys/node.keys

```diff
@@ -1,12 +1,9 @@
+4ED778F539E3634C779C87C6D7062848A1AB005C
 94AE36675C464D64BAFA68DD7434390BDBE9B9C5
-FD3A5288F042B6850C66B31F09FE44734EB7990E
 71DCFD284A79C3B38668286BC97EC7A07EDE3FC1
-DD8F2338BAE7501E3DD5AC78C273792F7D83545D
-C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8
-B9AE9905FFD7803F25714661B63B535A4C206CA9
-77984A986EBC2AA786BC0F66B01FBB92821C587A
 8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600
-4ED778F539E3634C779C87C6D7062848A1AB005C
+C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8
+C82FA3AE1CBEDC6BE46B9360C43CEC45C17AB93C
+DD8F2338BAE7501E3DD5AC78C273792F7D83545D
 A48C2BEE680E841632CD4E44F07496B3EB3C1762
 B9E2F5981AA6E0CD28160D9FF13993A75599653C
-C82FA3AE1CBEDC6BE46B9360C43CEC45C17AB93C
```

Comment:
Not sure the reason to change the order here 🤔

Reply:
It might make it easier to keep in sync with the list of keys in https://github.com/nodejs/node/#release-keys (where it's ordered by the name of the releaser the key belongs to).

Reply:
Should be some script ideally that dumps from that repo, sorts and inserts somewhere in this repo for consumption